packages/react-native/src/primitives/TextField/types.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  KeyboardTypeOptions,
+  StyleProp,
+  TextInputProps,
+  TextStyle,
+  ViewStyle,
+} from 'react-native';
+import { LabelProps } from '../Label/types';
+
+export interface TextFieldProps extends TextInputProps {
+  /**
+   * @description
```

Comment:
I like that you're annotating the props with comments. Do you think we should do this for the other primitives as well?

Reply:
Not in this PR though lol